user_docs/en/userGuide.t2t

```diff
@@ -3659,11 +3664,11 @@ Due to this, and to maintain compatibility with other screen readers in Taiwan,
 
 ++ Eurobraille displays ++[Eurobraille]
 The b.book, b.note, Esys, Esytime and Iris displays from Eurobraille are supported by NVDA.  
-These devices have a braille keyboard with 10 keys. 
+These devices have a Braille keyboard with 10 keys. 
 Of the two keys placed like a space bar, the left key is corresponding to the backspace key and the right key to the space key.
-Connected via USB, these devices have one stand-alone usb keyboard. 
-It is possible to enable/disable this keyboard with the checkbox ‘HID Keyboard simulation’ in braille setting panel. 
-The braille keyboard describes below is the braille keyboard when this checkbox is not checked.
+Connected via USB, these devices have one stand-alone USB keyboard. 
+It is possible to enable/disable this keyboard with the checkbox "HID Keyboard simulation" in Braille setting panel. 

```

Comment:
@FalkoBabbage - can you also clarify this section? we don't have a setting that matches this name/purpose in NVDA

Reply:
As far as I am aware, this can only be done through input gestures

Reply:
I've opened #15194 to address this

Reply:
> @FalkoBabbage - can you also clarify this section? we don't have a setting that matches this name/purpose in NVDA

I cannot verify now, but I think that the checkbox "HID Keyboard simulation" in Braille setting panel is added only when an Eurobraille device is detected.